### ISSUE TITLE
ARM Support

### ISSUE THREAD

**Issue by GergoEB** [Ym0T/pterodactyl-nginx-egg#10]
Hi,
Does this currently support arm as I am currently having no luck with it, it seems to be built for amd64, however i dont think there is anything stopping it from supporting arm too, other than running a slightly different command
Thanks

**Comment by Ym0T** [comment]
Hello, the egg was developed for amd64. I didn't even think about the arm support. 
I will have a look at the whole thing tonight to add support for arm maybe.

**Comment by GergoEB** [comment]
Thanks!

**Comment by GergoEB** [comment]
Hello,
I was right, it is very simple to do this.
I've done it on the weekend and have since tested it quite a bit, all that was required is giving dockerx 2 platforms (I used amd64 and arm64) then building. It takes twice as long but it adds support for both in a single image!
If interested, I used GitHub actions in a fork of this repo: https://github.com/GergoEB/pterodactyl-nginx-egg/blob/d937cc4ed30be9080673fcc93f2024218c708789/.github/workflows/docker-image.yml#L81
